### DISCUSSION TITLE
Adding a song to a playlist

### DISCUSSION THREAD

**Discussion by Taxicletter** [audacious-media-player/audacious#1384] [General]
Hi,
Is it correct that I cannot select "add to playlist" anywhere, when I selected a song in my library (or just any playlist where you can choose files)? You can only drag from the filemanager?

**Comment by radioactiveman** [comment]
I don't understand what you mean. Can you give more details or even better a screen recording of what you are trying to do?

**Comment by Taxicletter** [comment]
![afbeelding](https://github.com/audacious-media-player/audacious/assets/18621091/fb6f4ca6-783e-4b45-aef9-6fd27b0b52bf)

Imagine I want to add the selected song to playlist "Muziekje". How could I do that, without first searching the file and dragging it to "Muziekje"?

**Comment by radioactiveman** [comment]
1. Ctrl+C
2. Select other playlist tab
3. Ctrl+V

**Comment by Taxicletter** [comment]
OK, thank you.
That's not ideal, I would prefer the possibility to add it from the context-menu. If you scan through a long list, switching tabs is not convenient.
But good to know this is the way to do it.
